### PR TITLE
Jk/feature/apollo mutations

### PR DESCRIPTION
**Pull request by JeffKersting**
#### Purpose:
 > Check all that apply.
- [x] Feature
- [ ] Refactor
- [ ] Bug Fix 
- [x] Clean Up

### What does this PR do?
 > Clearly and concisely describe changes made. 

Adds update user mutation (GraphQL/update-user.js). Removes User-Landing state hooks and pulls data from apollo cache using a readQuery. useEffect also removed from User-Landing
Update user mutation call passes refetchQuery parameters to update user landing page on update (Update-Form line 49).
Updates Update-Form to take cached data and edits the prop types accordingly. 

Issues closed #

Because refetchQueries fires two queries in succession, there is a slight delay in the updated data being rendered on the page. Nothing significant, can revisit more efficient methods upon completion of MVP. 

> Describe your code changes in detail for reviewers.

All *changes* occur on the User-Landing and Update-Form components.
- On User-Landing, pretty much all react hooks have been removed (save for updating), and data is instead being pulled 
 directly from apollo (through readQuery). 
 - On Update-Form the bulk of the changes are in the initial state values being passed through from User-Landing, with the addition of the mutation being fired on submit. 

GraphQL/update-user
Update-Form
User-Landing

N/A (cypress/apollo stubbing)

### CODE REVIEW

File: src/GraphQL/update-user.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
+import { gql } from '@apollo/client'
+// import GET_USER from './queries'
+
+export default gql `
+  mutation ($id: ID!, $description: String!) {
```
Comment (gaj23) [inline src/GraphQL/update-user.js:6]: I think it'd be great to confirm with BE that this is the only expected mutation is the about me/description.

**Review by gaj23 — APPROVED** [review]
Great, thank you for these additions!

File: cypress/integration/test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  })
+  it('Should greet the user & describe app', () => {
+    cy.get('main h1').contains('welcome')
+    cy.get('main p').should('be.visible')
+  })
+  it('Should have a logged out version of nav bar & home', () => {
+  })
+  it('Should be able to login page and log in', () => {
+  })
+})
```
Comment (kelsiebesingeryeh) [inline cypress/integration/test.js:14]: this will be changed once we push up the testing branch

File: src/GraphQL/update-user.js (hunk 1 above)
Comment (kelsiebesingeryeh) [reply src/GraphQL/update-user.js:6]: agree

**Review by kelsiebesingeryeh — APPROVED** [review]

**Pull request merged**